cmd/kubeadm/app/util/config/common_test.go

```diff
@@ -124,11 +124,11 @@ func TestDetectUnsupportedVersion(t *testing.T) {
 			fileContents: bytes.Join([][]byte{files["Foo"], files["Master_v1alpha1"]}, []byte(constants.YAMLDocumentSeparator)),
 			expectedErr:  true,
 		},
-		// TODO: implement mustnotMix v1alpha3 v1beta1 after introducing v1beta1
+		// TODO: implement CanMix v1alpha3 v1beta1 after introducing v1beta1
 		{
-			name:         "MustNotMixInitJoin",
+			name:         "CanMixInitJoin",
```

Comment:
Could you put a comment above this test.  
To the casual reader this would be confusing.